From 9d8d30f437ca0b595ab08b64229dd98a5fd21832 Mon Sep 17 00:00:00 2001
From: Guenter Roeck <groeck@chromium.org>
Date: Mon, 17 Jun 2019 10:39:15 -0700
Subject: [PATCH] FIXUP: CHROMIUM: i915: intel_dp_aux_backlight: Fix max
 backlight calculations

Attempts to build an image with gcc fails as follows.

drivers/gpu/drm/i915/intel_dp_aux_backlight.c:
	In function 'intel_dp_aux_setup_backlight':
drivers/gpu/drm/i915/intel_dp_aux_backlight.c:274:19: error:
	unused variable 'intel_dp'

clang does not care about unused variables, but gcc does, and violently so.

BUG=b:134778235
TEST=Build image with "USE=-clang"

Change-Id: I230c0bbd3a138b3249ce9148927ee0ab3243972b
Signed-off-by: Guenter Roeck <groeck@chromium.org>
Reviewed-on: https://chromium-review.googlesource.com/c/chromiumos/third_party/kernel/+/1662628
Reviewed-by: Furquan Shaikh <furquan@chromium.org>
---
 drivers/gpu/drm/i915/intel_dp_aux_backlight.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_dp_aux_backlight.c b/drivers/gpu/drm/i915/intel_dp_aux_backlight.c
index 9fc898175d95..e500e8c5b031 100644
--- a/drivers/gpu/drm/i915/intel_dp_aux_backlight.c
+++ b/drivers/gpu/drm/i915/intel_dp_aux_backlight.c
@@ -271,7 +271,6 @@ static u32 intel_dp_aux_calc_max_backlight(struct intel_connector *connector)
 static int intel_dp_aux_setup_backlight(struct intel_connector *connector,
 					enum pipe pipe)
 {
-	struct intel_dp *intel_dp = enc_to_intel_dp(&connector->encoder->base);
 	struct intel_panel *panel = &connector->panel;
 
 	panel->backlight.max = intel_dp_aux_calc_max_backlight(connector);
-- 
2.23.0.rc1.153.gdeed80330f-goog

